docs/gitBranching.html

```diff
@@ -0,0 +1,268 @@
+                        code in any collaborative project setting. This method of project development is used constantly in any
+                        real-world development career in the coding, so making git branching and merging second-nature to you is a
+                        highly valuable skill. And it's very simple once you get used to it!</p>
+                    <p>Here, we will walk you through the philosophy of git branching and how to incorporate this essential workflow
+                        into your GitHub projects.</p>
+                </div>
+        <div>
+            
+            <p><strong>Table of Contents</strong></p>
+            <ul>
+                <li><a href="#the-basics-of-git-branching">The Basics of Git Branching</a></li>
+                <li><a href="#why-we-use-git-branches-explained-with-twinkies">Why We Use Git Branches, Explained with
+                        Twinkies</a></li>
+                <li><a href="#the-terminalbash-shell-commands--what-they-do">The Terminal/Bash Shell Commands &amp; What They
+                        Do</a></li>
+                <li><a href="#git-branch-basic-workflow">Git Branch Basic Workflow</a>
+                    <ul>
+                        <li><a href="#the-ultimate-daily-git-branch-checklist">The Ultimate Git Branch Daily Checklist</a></li>
+                        <li><a href="#keeping-your-branch-up-to-date-with-main-branch">Keeping your branch Up-To-Date with main
+                                branch</a></li>
+                        <li><a href="#pull-requests">Pull Requests</a></li>
+                        
+                    </ul>
+                </li>
+                <li><a href="#common-questions-and-answers">Common Questions and Answers</a>
+                    <ul>
+                        <li><a href="#what-is-an-upstream">What is an upstream?</a>
+                            <ul>
+                                <li><a href="#what-good-is-an-upstream">What good is an upstream?</a></li>
+                                <li><a href="#the-upstream-affects-git-merge-and-git-rebase-too">The upstream affects git merge
+                                        and git rebase too</a></li>
+                                <li><a href="#the-upstream-affects-git-pull">The upstream affects git pull</a></li>
+                                <li><a href="#the-upstream-affects-git-status">The upstream affects git status</a></li>
+                            </ul>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+        </div>
+        
+        <h1 id="why-we-use-git-branches-explained-with-twinkies">Why We Use Git Branches, Explained with Twinkies</h1>
+        <div class="slideWrap">
+            <img class="twinkie"
+                src="file:///c:\Users\18144\OneDrive\Documents\GitHub\newTutorials\miaGitTutorials\images\twinkieSlides\twinkie1.png"
```

Comment:
remove everything in the file path up until images directory. That information is redundant and could expose your C drive. Apply these changes to all photos in the image folder.